### DISCUSSION TITLE
No UART Activity On Bootup

### DISCUSSION THREAD

**Discussion by TrevorCash** [xjtuecho/EBAZ4205#58] [Troubleshooting]
Hi All,

I recieved my EBAZ4205.

I shorted D24.

When I power on the board with 12v on the Power Connector (J4) It uses about 110mA which seems resonable. 

However I am not getting any activity on the UART connection on boot. ( 115200baud, 8bit, 1 stop, no parity)

I also cannot program the board with my xilinx Jtag - it gives a "check board connections error".

When the board is powered on LED4 is lit constant green.  This is the only activity I see.

My main goal is to get JTAG programming working.

**Comment by DavidJRichards** [comment]
Greetings Trevor,
Is yours a stock board or has it been modified in any way? If it has the SDCard conversion then there will be no LED activity unless a card is inserted.

The serial output from the uart pins is 3v3 (not RS232), also be sure to connect Rx and Tx to the correct pins on your adaptor (try swapping if not sure)

You could check the power test points labelled on the PCB silk screen, I have seen boards with broken inductors or other parts knocked off the board.

You could ask for help on the Telegram forum, there is probably more activity there than here. https://t.me/EBAZ4205

Kind regards, David.

**Reply by TrevorCash** [reply]
Hi David,

Thanks for the reply,

I got the board booting now - and I can get into linux via the UART.  (I had shorted a resistor from another guide prematurely)

The Board looks well populated


The main issue for me now is the JTAG connectivity, Vivado gives warning:

```
 [Labtools 27-2269] No devices detected on target localhost:3121/xilinx_tcf/Digilent/210357A7D00EA.
Check cable connectivity and that the target board is powered up then
use the disconnect_hw_server and connect_hw_server to re-register this hardware target.
```

I am using a DLC9LP XILINX-IC

I will check on Telegram as well.

Thanks,
Trevor
